vm/gce/gce.go

```diff
@@ -138,6 +139,31 @@ func Ctor(env *vmimpl.Env, consoleReadCmd string) (*Pool, error) {
 	return pool, nil
 }
 
+const (
+	gceInitAttempts = 3
+	gceInitBackoff  = 5 * time.Second
+)
+
+func initGCE(zoneID string) (*gce.Context, error) {
+	// There happen some transient GCE init errors on and off.
+	// Let's try it several times before aborting.
+	var (
+		GCE *gce.Context
+		err error
+	)
+	for i := 1; i <= gceInitAttempts; i++ {
+		if i > 1 {
+			time.Sleep(gceInitBackoff)
+		}
+		GCE, err = gce.NewContext(zoneID)
+		if err == nil {
+			return GCE, nil
+		}
+		log.Logf(0, "init GCE attempt %d/%d failed: %v", i, gceInitAttempts, err)
```

Comment:
s/i/i+1/ so that it prints 1/3, 2/3, 3/3.

Reply:
The loop is `for i := 1; i <= gceInitAttempts; i++ {`, so it's not needed.